packages/wrangler/src/__tests__/deploy.test.ts

```diff
@@ -6104,11 +6077,13 @@ addEventListener('fetch', event => {});`
 
 		describe("[logfwdr]", () => {
 			it("should support logfwdr bindings", async () => {
-				fs.writeFileSync("./message.capnp.compiled", "compiled capnp messages");
+				(callCapnp as jest.MockedFunction<typeof callCapnp>).mockImplementation(
+					() => Buffer.from("compiled capnp messages")
+				);
```

Comment:
If possible, it would be nice to see the mocking here be slightly lower-level (on `child_process`, perhaps, as with `cloudflared`) so that the rest of the logic of `callCapnp()` can be exercised in tests

Reply:
I spent several hours trying to mock child process with Brendan - it causes a lot of issues with execa, and other usages of child_process in the (very large) test suite.

If you're able to successfully find a way to mock child process, which keeps the other tests happy, I'll definitely implement it!